core/app/models/spree/address.rb

```diff
@@ -5,7 +5,7 @@ class Address < Spree::Base
     belongs_to :country, class_name: "Spree::Country"
     belongs_to :state, class_name: "Spree::State"
 
-    validates :firstname, :lastname, :address1, :city, :country_id, presence: true
```

Comment:
Is this not going to potentially be a breaking change for some people? Did you find why this was required in the first place or is it one of those "historic" things?